x/tokenomics/keeper/proof_requirement_test.go

```diff
@@ -41,21 +44,19 @@ func TestKeeper_IsProofRequired(t *testing.T) {
 		isRequired, err := keepers.Keeper.IsProofRequiredForClaim(sdkCtx, &claim)
 		require.NoError(t, err)
 
-		samples[isRequired]++
+		switch isRequired {
+		case true:
+			numTrueSamples++
+		case false:
+			numFalseSamples++
```

Comment:
If you want to go even further, you can just keep a count of `numTrueSamples` and do `numFalseSamples = sampleSize - numTrueSamples` at the end